package/kernel/mac80211/ath.mk

```diff
@@ -352,6 +355,22 @@ This module adds support for Qualcomm Technologies 802.11ax family of
 chipsets with PCI bus.
 endef
 
+define KernelPackage/ath12k
+  $(call KernelPackage/mac80211/Default)
+  TITLE:=Qualcomm 802.11be wireless chipset support
+  URL:=https://wireless.wiki.kernel.org/en/users/drivers/ath12k
+  DEPENDS+= +kmod-ath +@DRIVER_11AC_SUPPORT +@DRIVER_11AX_SUPPORT @!LINUX_5_15 \
+  +@DRIVER_11BE_SUPPORT +kmod-crypto-michael-mic +kmod-qrtr-mhi \
```

Comment:
`DRIVER_11BE_SUPPORT` does not exists yet. 
If you want to add it here you should add the option too. you can look at the code for ax: 41d7a14ead50e391a7e94386911ab2e80616c72c